### ISSUE TITLE
When I try to initialize Foundation plugins in a Meteor.js app my app breaks

### ISSUE THREAD

**Issue by SamuelEarl** [zurb/foundation-sites#9027]
I am creating a Meteor app with Foundation. I am trying to initialize Foundation plugins globally on my main layout template, but whenever I add the onRendered() function block to my template helper my app disappears in the browser (i.e., the screen goes completely white). When I take the onRendered() block out, then my app reappears, but the Foundation plugins (e.g., tabs, accordions) don't work.

This is the code in my main layout template:

```
<template name="app-body">
  <body>
    {{> header}}
    {{> Template.dynamic template=content}}
    {{> footer}}
  </body>
</template>
```

This is the code in my template helper file:

```
import { Template } from 'meteor/templating';
import './app-body.html';

Template.app-body.onRendered(function () {
  $(document).foundation();
});
```

Any help you can offer would be greatly appreciated.

Thank you!

**Comment by Owlbertz** [comment]
I don't know much about Meteor (in terms of pre-compiling features), but as far as I know `Template.app-body` is not valid JavaScript. Have you tried `Template['app-body']`?
Otherwise, what happens if you put a simple `console.log()` into the callback and don't initialize Foundation there?

**Comment by SamuelEarl** [comment]
Thank you Owlbertz. I was able to figure out what the problem was based on your comment. I found out that I was using the wrong naming convention for my Meteor templates. I was using a hyphen to separate the words in the template name (e.g., app-body), but I should have been using either camelCase (e.g., appBody) or underscores (e.g., app_body) to separate words within template names. When I made that name change across my app, Foundation worked again. Thank you so much for your help, Owlbertz!
